core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -610,10 +610,10 @@ private void readCurrentBlockToPos(long pos) throws IOException {
       return;
     }
 
-    // Do not set the buffer size too small to avoid slowing down seek by too much.
     byte[] buffer = new byte[Math.min((int) mSeekBufferSizeBytes, (int) len)];
     do {
-      int bytesRead = read(buffer);
+      // Account for the last read which might be less than mSeekBufferSizeBytes bytes.
```

Comment:
should we keep `buffer` a method static member of size `mSeekBufferSizeByte`  rather than create a buffer each time?